docs/managed-datahub/upgrade_core_to_cloud.md

```diff
@@ -0,0 +1,262 @@
+# Upgrading from DataHub Core to DataHub Cloud
+
+Looking to upgrade to **DataHub Cloud**, but don't have an account yet? Start [here](https://datahub.com/demo/).
+
+Once you have a **DataHub Cloud** instance, you can seamlessly transfer all metadata from your self-hosted **DataHub Core** instance
+to **DataHub Cloud** using the DataHub CLI. In this guide, we'll show you how.
+
+## Prerequisites
+
+Before starting the upgrade process:
+
+1. **DataHub Cloud Account**: Ensure you have an active DataHub Cloud instance with an API token
+2. **Database Access**: You'll need read access to your DataHub Core MySQL or PostgreSQL database
+3. **DataHub CLI**: Install the DataHub CLI with `pip install acryl-datahub`
+4. **Network Connectivity**: Ensure your upgrade environment can access both your source database and DataHub Cloud
+5. **Database Index**: Verify that the `createdon` column is indexed in your source database:
```

Comment:
this should be done by default right? should we give instructions on how to verify? the instructions are just how to add it, not to verify presence